[script:site] An error occurred due to the '.DS_Store' file

<!-- generated by idux-issue-helper:en. DO NOT REMOVE -->
### Reproduction link

### Steps to reproduce
```
 /Volumes/Macintosh HD/project/IduxFE/idux [main] npm install
 /Volumes/Macintosh HD/project/IduxFE/idux [main] npm run start

> idux@0.0.0-NOT-USED start /Volumes/Macintosh HD/project/IduxFE/idux
> gulp start:dev

[15:25:20] Requiring external module ts-node/register
[15:25:32] Using gulpfile /Volumes/Macintosh HD/project/IduxFE/idux/gulpfile.ts
[15:25:32] Starting 'start:dev'...
[15:25:32] Starting 'icons:copy'...
[15:25:33] Finished 'icons:copy' after 806 ms
[15:25:33] Starting 'site:start'...
[15:25:33] Starting 'init:site'...
[15:25:33] 'init:site' errored after 20 ms
[15:25:33] Error: ENOTDIR: not a directory, scandir '/Volumes/Macintosh HD/project/IduxFE/idux/packages/.DS_Store'
    at Object.readdirSync (fs.js:1021:3)
    at /Volumes/Macintosh HD/project/IduxFE/idux/scripts/gulp/site/index.ts:35:5
    at Array.forEach (<anonymous>)
    at Object.initSite (/Volumes/Macintosh HD/project/IduxFE/idux/scripts/gulp/site/index.ts:32:28)
    at /Volumes/Macintosh HD/project/IduxFE/idux/scripts/gulp/tasks/site.ts:9:3
    at init:site (/Volumes/Macintosh HD/project/IduxFE/idux/node_modules/undertaker/lib/set-task.js:13:15)
    at bound (domain.js:413:15)
    at runBound (domain.js:424:12)
    at asyncRunner (/Volumes/Macintosh HD/project/IduxFE/idux/node_modules/async-done/index.js:55:18)
    at processTicksAndRejections (internal/process/task_queues.js:75:11)
[15:25:33] 'site:start' errored after 106 ms
[15:25:33] 'start:dev' errored after 1 s
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! idux@0.0.0-NOT-USED start: `gulp start:dev`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the idux@0.0.0-NOT-USED start script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
```
### What is expected?

`npm run start` success

### What is actually happening?

There is a hidden ‘.DS_Store’ file in the MAC, which leads to an error in the startup script

### Any additional comments? (optional)

---
| Environment | Info |
|---|---|
| @idux |  main  |
| Browser |  ALL  |

--- a/scripts/gulp/site/index.ts
+++ b/scripts/gulp/site/index.ts
@@ -30,8 +30,11 @@ export function initSite(): void {
 
   const filterComponentName = ['style', 'core', 'i18n', 'version', 'utils']
   readdirSync(packageRoot).forEach(packageName => {
-    docsMeta[packageName] = {}
     const packageDirname = join(packageRoot, packageName)
+    if (!statSync(packageDirname).isDirectory()) {
+      return
+    }
+    docsMeta[packageName] = {}
     readdirSync(packageDirname).forEach(componentName => {
       if (filterComponentName.includes(componentName)) {
         return

